docs/dev_guide/contents/extending_fido.rst

```diff
@@ -297,7 +297,7 @@ The ``search()`` method has the job of taking a set of user queries and returnin
 The general flow of a ``search()`` method is:
 
 * Call your instance of an `.AttrWalker` to convert the input into a form expected by your API.
-* Make as many requests to your API as needed to fulfill the query. (Generally one per element of the outer `.AttrOr`).
+* Make as many requests to your API as needed to fulfill the query. (Generally one per element of the outer `sunpy.net.attrs.AttrOr`).
```

Comment:
There are several other places where `sunpy.net.attr.AttrOr` is used, we should update it to `sunpy.net.attrs.AttrOr` where possible. The same goes for the `And` version.